Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -123,6 +123,8 @@ class TabDisplayManager: NSObject {
         isPrivate = isOn
         UserDefaults.standard.set(isPrivate, forKey: "wasLastSessionPrivate")
 
+        UnifiedTelemetry.recordEvent(category: .action, method: .tap, object: .privateBrowsingButton)
```

Comment:
Should we also record whether it is turning on or off?

Reply:
Good idea, added that